tests/other/test_split_module_ex.post.js

```diff
@@ -0,0 +1,17 @@
+function saveProfileData() {
+  var __write_profile = Module['asm']['__write_profile'];
+  if (__write_profile) {
+    var len = __write_profile(0, 0);
+    var offset = _malloc(len);
+    var actualLen = __write_profile(offset, len);
+    var profile_data = new Uint8Array(buffer, offset, len);
+    nodeFS.writeFileSync('profile.data', profile_data);
+    console.log('wrote profile of', actualLen, 'bytes (allocated', len, 'bytes)');
+    _free(offset);
+  }
+
+  // Say hello *after* recording the profile so that all functions are deferred.
```

Comment:
If nothing has run yet, won't the profile be empty?   Can we instead just remove this code and not write a profile at all?

Reply:
my bad. I couldn't verify profile data earlier. I have fixed the training and verified that profile data has some functions captured:
`profile.data:hash=10519562697473171297:hot=176:all fn=678`

PS: this was dumped by using portion of binaryen code to read info from profile data